framework/tools/generators.py

```diff
@@ -14,3 +19,45 @@ def generate_string(length: int, additional_characters: list = None) -> str:
         result += additional_characters
 
     return "".join(result)
+
+
+def generate_password(length: int) -> str:
+    """
+     Function generates and returns random generated password.
+     Args:
+         length (int): length of password
+    """
```

Comment:
```
    """Function generates and returns random generated password.

    Args:
         length: length of password
    """
```
it seems to me that there are more indents next to Args than 4 spaces. You can do it in this form (I don't see the point of adding int in parentheses, because you already use typing)

Reply:
Fixed